docs/core/tutorials/index.md

```diff
@@ -41,6 +41,8 @@ The following tutorials are available for learning about .NET Core.
 - [Unit testing with MSTest and .NET Core](../testing/unit-testing-with-mstest.md)
 - [Developing Libraries with Cross Platform Tools](libraries.md)
 - [Hosting .NET Core from native code](netcore-hosting.md)
-- [Create a custom template for dotnet new](cli-templates-create-item-template.md)
+- [Create a custom item template for the CLI](cli-templates-create-item-template.md)
```

Comment:
wondering if this was like this because they need to be done in sequence @Thraka?
If so, maybe we should use the node name here instead?

Reply:
These three are a sequential tutorial series, so you only need to do the first.